medical_prescription/views/medical_prescription_order_line_view.xml

```diff
@@ -0,0 +1,71 @@
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='dose_unit_id']" position="after">
+                    <field name="prescription_order_id" />
+                </xpath>
+                <xpath expr="//filter[@name='frequency_prn']" position="after">
+                    <filter name="is_substitutable" string="Is Substitutable"
+                        domain="[('is_substitutable', '=', True)]"/>
+                </xpath>
+                <xpath expr="//group" position="inside">
+                    <filter string="Prescription Order"
+                        domain="[]" context="{'group_by':'prescription_order_id'}" />
+                    <filter string="Medication"
+                        domain="[]" context="{'group_by':'medical_medication_id'}" />
+                </xpath>
+            </field>
+        </record>
+
+        <record id="medical_prescription_order_line_tree_view" model="ir.ui.view">
+            <field name="name">medical.prescription.order.line.tree</field>
+            <field name="model">medical.prescription.order.line</field>
+            <field name="mode">primary</field>
+            <field name="inherit_id"
+                ref="medical_medication.medical_patient_medication_tree_view"/>
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='patient_id']" position="before">
+                    <field name="prescription_order_id" />
+                </xpath>
+            </field>
+        </record>
+
+        <record id="medical_prescription_order_line_form_view" model="ir.ui.view">
+            <field name="name">medical.prescription.order.line.form</field>
+            <field name="model">medical.prescription.order.line</field>
+            <field name="mode">primary</field>
+            <field name="inherit_id"
+                ref="medical_medication.medical_patient_medication_form_view"/>
+            <field name="arch" type="xml">
+                <xpath expr="//field[@name='patient_id']" postition="after">
+                    <field name="prescription_order_id" />
+                    <field name="is_substitutable" />
+                </xpath>
+            </field>
+        </record>
+
+        <record model="ir.actions.act_window" id="medical_prescription_order_line_action">
+            <field name="name">Prescription Order Lines</field>
+            <field name="type">ir.actions.act_window</field>
+            <field name="res_model">medical.prescription.order.line</field>
+            <field name="view_type">form</field>
+            <field name="view_mode">tree,form</field>
+            <field name="search_view_id" ref="medical_prescription_order_line_search_view" />
+            <field name="view_id" ref="medical_prescription_order_line_tree_view" />
+            <field name="domain">[]</field>
+            <field name="context">{}</field>
+        </record>
+
+        <menuitem id="medical_prescription_order_line_menu" sequence="20"
```

Comment:
Hi @lasley 
You're right about the real necessity of this menu (as there already is a one2many into the prescription_order)  but I don't think it is disturbing as it offers a quicks access on lines (a litle bit like timesheet and timesheet line into native oddo addons)
Concerning your point of view about adding a prescription_id into the form view of order line, it's already the case (except I called it prescription_order_id). But to create prescription I suggest users to firstly created a prescription order and then click on `add an item` in order to use the power of the temple to allow user to quickly add prescription order line)